articles/azure/azs-how-use-azure-api.md

```diff
@@ -159,7 +189,7 @@ where:
 
 For example, the following API call returns information about region health:
 
-`https://management.local.azurestack.external/subscriptions/800c4168-3eb1-405b-a4ca-919fe7ee42e9/resourcegroups/system.local/providers/microsoft.infrastructureinsights.admin/regionhealths/local/Alerts?$filter=(Properties/State eq 'Active') and (Properties/Severity eq 'Critical')&$orderby=Properties/CreatedTimestamp desc&api-version=2016-05-01"`
+`https://adminmanagement.frn00006.azure.ukcloud.com/subscriptions/800c4168-3eb1-405b-a4ca-919fe7ee42e9/resourcegroups/system.local/providers/microsoft.infrastructureinsights.admin/regionhealths/local/Alerts?$filter=(Properties/State eq 'Active') and (Properties/Severity eq 'Critical')&$orderby=Properties/CreatedTimestamp desc&api-version=2016-05-01"`
```

Comment:
As this is the user guide for the Azure Stack API can we change this to be something for the user api? Something like listing all the virtual machines.